crates/polars-plan/src/dsl/python_udf.rs

```diff
@@ -57,14 +57,14 @@ impl PythonUdfExpression {
         // Handle pickle metadata
         let use_cloudpickle = buf[0];
         if use_cloudpickle != 0 {
```

Comment:
I'm not sure about this, but we apparently don't enforce the Python version equality if we are using cloudpickle